README.md

```diff
@@ -2,10 +2,56 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+It is the mission of the Node.js Foundation and the TSC to provide a home for 
+openly-governed and community-managed open source projects that are focused on 
+supporting and providing benefit to the Node.js ecosystem.
```

Comment:
> To "enable and encourage the adoption" isn't actionable enough.

That's because it's a mission statement, not an action plan. If we try to capture the entire plan in one sentence we are bound to grossly oversimplify it, especially at this early stage. For example the fact that a project is "focused on supporting and providing benefit to the Node.js ecosystem" doesn't mean that it is of *strategic* importance to the ecosystem, but this statement make accepting all of those projects *the only* mission of the TSC, regardless of their strategic importance. 

I am not saying that the TSC shouldn't host projects beyond core. But I think that it shouldn't jump to hosting just *any* project that needs a home either, at least without a controlled learning phase and not before having clarified the criteria. Until those criteria are defined and the TSC feels comfortable about them, I would prefer a more vague mission statement to a misleading one.